zero/gkyl_gk_geometry_mapc2p.h

```diff
@@ -13,15 +13,17 @@
  * @param bmag function which gives |B| in computational space
  * @param bmag_ctx Context for calculating |B|
  */
-struct gk_geometry* gkyl_gk_geometry_mapc2p_new(const struct gkyl_rect_grid* grid, const struct gkyl_range *range, const struct gkyl_range* range_ext, 
-  const struct gkyl_basis* basis, evalf_t mapc2p_func, void* mapc2p_ctx, evalf_t bmag_func, void* bmag_ctx, bool use_gpu);
+struct gk_geometry* gkyl_gk_geometry_mapc2p_new(const struct gkyl_rect_grid* grid, const struct gkyl_range *range,
+    const struct gkyl_range* range_ext, const struct gkyl_range *global, const struct gkyl_range* global_ext,
+    const struct gkyl_basis* basis, evalf_t mapc2p_func, void* mapc2p_ctx, evalf_t bmag_func, void* bmag_ctx, bool use_gpu);
 
 /**
  * Create a new gk geometry object that lives on NV-GPU: see new() method
  * above for documentation.
  */
 
-struct gk_geometry* gkyl_gk_geometry_mapc2p_cu_dev_new(const struct gkyl_rect_grid* grid, const struct gkyl_range *range, const struct gkyl_range* range_ext, 
-  const struct gkyl_basis* basis, evalf_t mapc2p_func, void* mapc2p_ctx, evalf_t bmag_func, void* bmag_ctx);
+struct gk_geometry* gkyl_gk_geometry_mapc2p_cu_dev_new(const struct gkyl_rect_grid* grid, const struct gkyl_range *range,
```

Comment:
I would recommend formatting these to have the two sets of ranges on new lines (so a new line for the local and local_ext and a new line for the global and global_ext), that way you don't have any big monster lines.